stable/joomla/templates/deployment.yaml

```diff
@@ -20,15 +20,28 @@ spec:
         app: {{ template "joomla.fullname" . }}
         chart: {{ template "joomla.chart" . }}
         release: {{ .Release.Name | quote }}
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotations` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?